tests/system/test_blob.py

```diff
@@ -97,6 +97,10 @@ def test_large_file_write_from_stream_w_failed_checksum(
     assert not blob.exists()
 
 
+@pytest.mark.skipif(
+    "API_ENDPOINT_OVERRIDE" in os.environ,
```

Comment:
Replace with boolean constant at top of file as discussed, thank you for the idea.